engine/rpcserver.go

```diff
@@ -2259,12 +2246,7 @@ func (s *RPCServer) GetExchangeTickerStream(r *gctrpc.GetExchangeTickerStreamReq
 			return errDispatchSystem
 		}
 
-		d := *data.(*interface{})
-		if d == nil {
-			return errors.New("unable to type assert data")
-		}
-
-		t, ok := d.(ticker.Price)
+		t, ok := data.(ticker.Price)
```

Comment:
go run ./... getexchangetickerstream -exchange="btc markets"
2022/04/14 12:25:45 rpc error: code = Unknown desc = unable to type assert ticker data
exit status 1